contents/docs/experiments/no-code-web-experiments.md

```diff
@@ -4,7 +4,10 @@ sidebar: Docs
 showTitle: true
 ---
 
-No-code web experiments enable you to run A/B tests, multivariate tests, and other experiments that modify your website without writing a single line of code.  
+> 🚧 **NOTE:** No-code web experiments are currently considered in `beta`. To access them, enable the [feature preview](https://us.posthog.com#panel=feature-previews) in your PostHog account. We are keen to gather as much feedback as possible so if you try this out please let us know. You can send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Aexperiments%3Alow) or one of our other [support options](/docs/support-options).
```

Comment:
idea: just add your email here and see who emails you?

Reply:
@Lior539 To clarify, are you suggesting I initiate the support contact form so it emails me directly, or simply link my email address here?

Reply:
Im suggesting you put your email here. The hypothesis being is that its more personal, so youre more likely to get quality feedback

Reply:
@Lior539 Sounds good! Updated with b81580a04